usmqe_tests/api/ceph/test_pool.py

```diff
@@ -0,0 +1,587 @@
+        valid_cluster_id,
+        invalid_pool_id,
+        valid_session_credentials):
+    """@pylatest api/ceph.delete_pool_invalid
+        API-ceph: delete_pool
+        ******************************
+
+        :authors:
+            - fbalak@redhat.com
+            - mkudlej@redhat.com
+
+        Description
+        ===========
+        Negative Delete from CRUD for pools.
+
+        Delete ceph pool ``invalid_pool_id`` via API.
+
+        .. test_step:: 1
+
+                Connect to Tendrl API via DELETE request
+                to ``APIURL/:cluster_id/CephDeletePool``
+                Where cluster_id is set to predefined value.
+
+        .. test_result:: 1
+
+                Server should return response in JSON format:
+
+                Return code should be **200** with data ``{"job_id":"_id_"}``.
+                And then joib should fail.
+                """
+
+    api = cephapi.TendrlApiCeph(auth=valid_session_credentials)
+    job_id = api.delete_pool(valid_cluster_id, invalid_pool_id)["job_id"]
+    api.wait_for_job_status(job_id, status="failed")
+
+
+def test_delete_pool_valid(
+        valid_cluster_id,
+        valid_pool_id,
+        valid_pool_name,
+        valid_session_credentials):
+    """@pylatest api/ceph.delete_pool
+        API-ceph: delete_pool
+        ******************************
+
+        :authors:
+            - fbalak@redhat.com
+            - mkudlej@redhat.com
+
+        Description
+        ===========
+        Positive Delete from CRUD for pools.
+
+        Delete ceph pool ``valid_pool_id`` via API.
+
+        .. test_step:: 1
+
+                Connect to Tendrl API via POST request
+                to ``APIURL/:cluster_id/CephDeletePool``
+                Where cluster_id is set to predefined value.
+
+        .. test_result:: 1
+
+                Server should return response in JSON format:
+
+                Return code should be **200** with data ``{"job_id":"_id_"}``.
+                And then job should finish.
+                """
+
+    api = cephapi.TendrlApiCeph(auth=valid_session_credentials)
+    job_id = api.delete_pool(valid_cluster_id, valid_pool_id)["job_id"]
+    LOGGER.info("Delete pool job_id: {}".format(job_id))
+    api.wait_for_job_status(job_id)
+    """@pylatest api/ceph.delete_pool
+        API-ceph: delete_pool
+        ******************************
+
+        :authors:
+            - fbalak@redhat.com
+            - mkudlej@redhat.com
+
+        Description
+        ===========
+
+        Check if there is not deleted pool in ceph cluster via CLI.
+
+        .. test_step:: 2
+
+            Connect to ceph monitor machine via ssh and run
+            ``ceph --cluster *clustername* pool status``
+
+        .. test_result:: 2
+
+            There should not be listed ceph pool named ``valid_pool_name``.
+
+            """
+    storage = ceph_cluster.CephCluster(pytest.config.getini("usm_ceph_cl_name"))
+
+    pytest.check(valid_pool_name not in storage.osd.pool_ls(),
```

Comment:
for sure there will not be, if the name was changed.
For update/delete the name should/could be taken from API.